[2024-06-16T10:42:56.769+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-16T10:42:56.779+0200] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: datalake_dag.format_carbon_data manual__2024-06-16T08:42:38.879209+00:00 [queued]>
[2024-06-16T10:42:56.782+0200] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: datalake_dag.format_carbon_data manual__2024-06-16T08:42:38.879209+00:00 [queued]>
[2024-06-16T10:42:56.782+0200] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-06-16T10:42:56.872+0200] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): format_carbon_data> on 2024-06-16 08:42:38.879209+00:00
[2024-06-16T10:42:56.874+0200] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-06-16T10:42:56.874+0200] {standard_task_runner.py:63} INFO - Started process 13910 to run task
[2024-06-16T10:42:56.877+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'datalake_dag', 'format_carbon_data', 'manual__2024-06-16T08:42:38.879209+00:00', '--job-id', '156', '--raw', '--subdir', 'DAGS_FOLDER/datalake_dag.py', '--cfg-path', '/tmp/tmpovc6o9lf']
[2024-06-16T10:42:56.878+0200] {standard_task_runner.py:91} INFO - Job 156: Subtask format_carbon_data
[2024-06-16T10:42:56.907+0200] {task_command.py:426} INFO - Running <TaskInstance: datalake_dag.format_carbon_data manual__2024-06-16T08:42:38.879209+00:00 [running]> on host Desktop-Hector.
[2024-06-16T10:42:56.956+0200] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='datalake_dag' AIRFLOW_CTX_TASK_ID='format_carbon_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-16T08:42:38.879209+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-16T08:42:38.879209+00:00'
[2024-06-16T10:42:56.957+0200] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-16T10:42:56.962+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-16T10:42:56.962+0200] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/hectorv/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/hectorv/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/hectorv/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/hectorv/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/hectorv/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/hectorv/airflow/dags/lib/format_ademe_data.py", line 36, in format_carbon_data
    sc = SparkContext('local')
  File "/home/hectorv/.local/lib/python3.10/site-packages/pyspark/context.py", line 201, in __init__
    SparkContext._ensure_initialized(self, gateway=gateway, conf=conf)
  File "/home/hectorv/.local/lib/python3.10/site-packages/pyspark/context.py", line 449, in _ensure_initialized
    raise ValueError(
ValueError: Cannot run multiple SparkContexts at once; existing SparkContext(app=pyspark-shell, master=local) created by __init__ at /home/hectorv/airflow/dags/lib/format_digital_iservices_data.py:36 
[2024-06-16T10:42:56.964+0200] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=datalake_dag, task_id=format_carbon_data, execution_date=20240616T084238, start_date=20240616T084256, end_date=20240616T084256
[2024-06-16T10:42:56.979+0200] {standard_task_runner.py:110} ERROR - Failed to execute job 156 for task format_carbon_data (Cannot run multiple SparkContexts at once; existing SparkContext(app=pyspark-shell, master=local) created by __init__ at /home/hectorv/airflow/dags/lib/format_digital_iservices_data.py:36 ; 13910)
[2024-06-16T10:42:57.009+0200] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-16T10:42:57.020+0200] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-16T10:42:57.031+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
